programs/nixfmt-classic.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, pkgs, config, ... }:
+let
+  cfg = config.programs.nixfmt-classic;
+in
+{
+  options.programs.nixfmt-classic = {
+    enable = lib.mkEnableOption "nixfmt-classic";
```

Comment:
> nixfmt was renamed to nixfmt-classic. The nixfmt attribute may be used for the new RFC 166-style formatter in the future, which is currently available as nixfmt-rfc-style

Renaming nixfmt -> nixfmt-classic is a breaking change.
cc @SebTM who introduced nixfmt linter.

Reply:
Given that both nixfmt-classic and nixfmt-rfc-style are CLI compatible, and that nixfmt-classic will probably disappear, I propose:
1. Keep `nixfmt` as the module name
2. Use `nixfmt-rfc-style` as the default package.
3. Deprecate the `nixfmt-rfc-style` module.
And if people want to use `nixfmt-classic`, they can set it as the package.

Reply:
Added a warning to `nixfmt-rfc-style`, set nixfmt default package to `nixfmt-rfc-style`.